saleor/graphql/schema.graphql

```diff
@@ -17591,6 +17591,9 @@ input OrderFulfillInput {
 
   """If true, then allow proceed fulfillment when stock is exceeded."""
   allowStockToBeExceeded: Boolean = false
+
```

Comment:
Is it a really good place for this parameter?
`OrderFulfill` allows you to create multiple different fulfillments (based on warehousePK). If we add this input here, no matter how many fulfillments we will create, all of them will have the same tracking number. (And it could not be true, as each fulfillment comes from a different warehouse).
Example. 
In the input, we provide `line: {Juce , quantity:1, warehouseID:13}`, and the second `line: {Apple, quanity:2, warehouse:33}`. This will create two separate fulfillments, and both of them will get the same `trackingNumber`. (But they came from different warehouses). 
cc @maarcingebala @jakubkuc @patrys

Reply:
It can be changed to be passed inside lines ( in `OrderFulfillLineInput` ) so that every separate fulfillment from line has  its own tracking number.

Reply:
@korycins That’s expected. There is no per-warehouse input and we can’t have a separate tracking number for each line.